src/Products/PloneMeeting/vocabularies.py

```
@@ -3467,3 +3492,927 @@ def __call__(self, context, only_active=False):
+
+# ============================================================================
+# MeetingConfig vocabulary factories
+# (delegates to MeetingConfig list* methods for config-dependent vocabularies)
+# ============================================================================
+
+def _dl_to_vocabulary(display_list):
+    """Convert an AT DisplayList to a zope.schema SimpleVocabulary."""
+    terms = []
+    for value in display_list:
+        label = display_list.getValue(value)
+        if not isinstance(value, basestring):
```

Ruff (0.15.11)

[error] Undefined name `basestring`

(F821)